draft-ietf-ppm-dap.md

```diff
@@ -1305,45 +1509,70 @@ given batch interval, denoted `batch_interval`, decryption works as follows:
 
 ~~~
 agg_share = OpenBase(enc_share.enc, sk, "dap-01 aggregate share" ||
-    server_role || 0x00, task_id || batch_interval, enc_share.payload)
+    server_role || 0x00, task_id || batch_selector, enc_share.payload)
 ~~~
 
 where `sk` is the HPKE secret key, `task_id` is the task ID for the collect
 request, and `server_role` is the role of the server that sent the aggregate
-share (`0x02` for the leader and `0x03` for the helper).
+share (`0x02` for the leader and `0x03` for the helper). The value of
+`batch_selector` is computed by the Collector from its query and the response to
+its query:
 
-### Validating Batch Parameters {#batch-parameter-validation}
+* For time-series tasks, the batch selector is the batch interval specified in
+  the query.
+
+* For fixed-chunk tasks, the batch selector is the chunk ID assigned sent in the
+  response.
+
+### Batch Validation {#batch-validation}
 
 Before an aggregator responds to a collect request or aggregate-share request,
 it must first check that the request does not violate the parameters associated
 with the DAP task. It does so as described here.
 
-First the aggregator checks that the request's batch interval respects the
-boundaries defined by the DAP task's parameters. Namely, it checks that both
-`batch_interval.start` and `batch_interval.duration` are divisible by
-`min_batch_duration` and that `batch_interval.duration >= min_batch_duration`.
-Unless both these conditions are true, the aggregator MUST abort and alert its
-peer with error "batchInvalid".
-
-Next, the aggregator checks that the request respects the generic privacy
-parameters of the DAP task. Let `X` denote the set of reports for which the
-aggregator has recovered a valid output share and which fall in the batch
-interval of the request.
-
-* If `len(X) < min_batch_size`, then the aggregator MUST abort and alert its
-  peer with "insufficientBatchSize".
-* The aggregator keeps track of the number of times each report was added to the
-  batch of an AggregateShareReq. If any report in `X` was added to at least
-  `max_batch_lifetime` previous batches, then the aggregator MUST abort and
-  alert the peer with "batchLifetimeExceeded".
-
-Finally, the aggregator checks that the batch interval defined by the collect
-request satisfies one of the conditions:
-
-1. The batch interval does not overlap with the batch interval of any prior
-   completed collect requests.
-1. The batch interval, including its start and duration values, match a prior
-   completed collect request.
+Frist the aggregator checks that the request respects any batch "boundaries"
```

Comment:
Maybe "boundary" is the wrong word? For time-series tasks, these are  the batch windows. For fixed-chunk tasks, these are the chunks.